src/errors/test-run/index.js

```diff
@@ -188,6 +189,7 @@ export class ExternalAssertionLibraryError extends TestRunErrorBase {
         super(TEST_RUN_ERRORS.externalAssertionLibraryError);
 
         this.errMsg   = String(err);
+        this.diff     = diff.generateDiff(diff.stringify(err.actual), diff.stringify(err.expected));
```

Comment:
I propose to rename the `generateDiff` function to `generate` and do not export the `stringify` function because it isn't used in other places.
```js
this.diff = diff.generate(err.actual, err.expected);
```